Sources/MockoloFramework/Templates/FunctionSuffixTemplate.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Copyright (c) 2018. Uber Technologies
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+extension FunctionSuffixClause {
+    func applyFunctionSuffixTemplate(
+        forClosureTemplate: Bool
```

Comment:
<img src="https://img.shields.io/badge/review--note-gray" alt="review-note"> 

For closure, `rethrows` has to be replaced with `throws`.

example: 

```swift
var barHandler: () throws -> Void
func foo(bar: () throws -> Void) rethrows -> Void {}